www/includes/easyparliament/templates/html/donate/_stripe_donate.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+# define array of payment amounts and current/one-off options
+# note - new annual and monthly payments need to be defined as new prices in stripe
+# e.g. 'donate_monthly_10'. 
+$payment_amounts = array(
+  'monthly' => array(
+    '5' => '£5',
+    '10' => '£10',
+    '25' => '£25',
+  ),
+  'annually' => array(
+    '10' => '£10',
+    '50' => '£50',
+    '100' => '£100',
+  ),
+  'one-off' => array(
+    '20' => '£20',
+    '50' => '£50',
+    '100' => '£100',
+  ),
+);
+
+$default_amounts = array(
+  'monthly' => '10',
+  'annually' => '10',
+  'one-off' => '50',
+);
+
+$default_type = 'annually';
+
+# use the how-often parameter if set, if not default to option at end of line (options are 'monthly', 'annually', or 'one-off')
+$initial_payment_type = isset($_GET['how-often']) ? $_GET['how-often'] : $default_type;
+
+# use the how-much parameter if set, if not default to default amount for initial payment type
+$how_much = isset($_GET['how-much']) ? $_GET['how-much'] : $default_amounts[$initial_payment_type];
+
+# if how-much is not in the allowed values for the current payment type, set to 'other', and set $other_how_much to the value of how-much
+if (!array_key_exists($how_much, $payment_amounts[$initial_payment_type])) {
+  $how_much = 'other';
+  $other_how_much = $_GET['how-much'];
+} else {
+  $other_how_much = '';
+}
+
+?>
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+
+
+<form class="donate-form" method="post" name="donation_form">
+
+    <div class="donate-form__error-wrapper">
+      <?php if (isset($error) and $error) { ?>
```

Comment:
Don't think $error is ever set by anything?

Reply:
I think a legacy from the older paypal forms, etc that may have refreshed the page rather than using ajax. Removing.